tcpdf.php

```diff
@@ -18885,7 +18906,11 @@ protected function openHTMLTagHandler($dom, $key, $cell) {
 					// data stream
 					$imgsrc = '@'.base64_decode(substr($imgsrc, 1));
 					$type = '';
-				} else {
+				} elseif ( $this->allowLocalFiles && substr($imgsrc, 0, 7) === 'file://') {
+                    // get image type from a local file path
+                    $imgsrc = substr($imgsrc, 7);
+                    $type = TCPDF_IMAGES::getImageFileType($imgsrc);
+                } else {
```

Comment:
```suggestion
				} elseif ( $this->allowLocalFiles && substr($imgsrc, 0, 7) === 'file://') {
                                    // get image type from a local file path
                                   $imgsrc = substr($imgsrc, 7);
                                    $type = TCPDF_IMAGES::getImageFileType($imgsrc);
                                } else {
```
Using an IDE is would be more precise ;)

Reply:
yeah,.. your change looks wrong too. should be tabs instead of spaces.

Reply:
Yeah I understand